Src/library/net/Clib/network.c

```diff
@@ -992,6 +1114,90 @@ EIF_INTEGER c_set_sock_opt_linger(EIF_INTEGER fd, EIF_BOOLEAN flag, EIF_INTEGER
 	return (EIF_INTEGER) result;
 }
 
+/********************************************************/
+/* Socket recv and send timeout options 				*/
+/********************************************************/
+
+EIF_INTEGER c_get_sock_recv_timeout(EIF_INTEGER fd, EIF_INTEGER level)
```

Comment:
It was to gather all related external together .. at first I added them to Misc 
because I was worry about platform other than Windows and Unices.